app/main/posts/views/share/share-menu.html

```diff
@@ -0,0 +1,78 @@
+<div class="modal-body">
+    <form class="panel-body" name="form" ng-submit="submit()">
+				<div class="listing init">
+
+						<div class="listing-item">
+								<div class="listing-item-primary">
+										<div class="listing-item-image">
+												<img class="icon" src="../../img/facebook.png" alt="">
+										</div>
+										<h2 class="listing-item-title">
+											<div class="fb-share-button"
+												data-href="{{shareUrl}}"
+												data-layout="button_count"
+												data-size="large"
+												data-mobile-iframe="true">
+													<a class="fb-xfbml-parse-ignore"
+														target="_blank"
+														ng-href="https://www.facebook.com/sharer/sharer.php?u={{shareUrlEncoded}}&amp;src=sdkpreparse">
+															Facebook
+													</a>
+											</div>
+										</h2>
+								</div>
+						</div>
+
+						<div class="listing-item">
+								<div class="listing-item-primary">
+										<div class="listing-item-image">
+												<img class="icon" src="../../img/twitter.png" alt="">
+										</div>
+										<h2 class="listing-item-title"><a ng-href="https://twitter.com/home?status={{shareUrl}}">Twitter</a></h2>
```

Comment:
Actually for twitter it appears to be ok unencoded, i'll test with encoded and see if they parse